File: table.c
#include <stdio.h>
int main()
{
int n;int i;
printf("Enter the number whose table is required : ");
scanf("%d", &n);
for(i=1;i<=10;i++)
printf("%d X %d = %d\n", n,i,n*i);
return 0;    
}

File: Day 1/table.java
import java.util.*;

class table {
    public static void main() {
        int n;
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter the number to print its table");
        n = sc.nextInt();
        for (int i = 1; i <= 10; i++) {
            System.out.println(n + " X " + i + " = " + (n * i));
        }
        sc.close();
    }
}